common/dynamicconfig/constants.go

```diff
@@ -4073,6 +4081,11 @@ var BoolKeys = map[BoolKey]DynamicBool{
 		Description:  "TaskSchedulerEnableRateLimiterShadowMode indicates whether the task scheduler rate limiter is in shadow mode",
 		DefaultValue: true,
 	},
+	TaskSchedulerEnableMigration: {
+		KeyName:      "history.taskSchedulerEnableMigration",
+		Description:  "TaskSchedulerEnableMigration indicates whether the task scheduler migration is enabled",
+		DefaultValue: false,
+	},
```

Comment:
do we need this extra bool flag? could `TaskSchedulerMigrationRatio: 0` indicate it's disabled?